pkg/fuzzer/queue/queue.go

```diff
@@ -172,6 +208,14 @@ func (r *Result) Stop() bool {
 	}
 }
 
+// Globs returns result of RequestTypeGlob.
+func (r *Result) GlobFiles() []string {
+	if len(r.Output) == 0 {
+		return nil
+	}
+	return strings.Split(string(r.Output), "\000")
```

Comment:
Just in case: can an empty last element (if every line is followed by `\0`, there must be some) be problematic for the further code?